endpoint/domain_filter_test.go

```diff
@@ -429,7 +429,7 @@ func TestDomainFilterIsConfigured(t *testing.T) {
 		{
 			[]string{"", ""},
 			[]string{""},
-			true,
+			false,
```

Comment:
This changes the behavior completely. Was the previous one wrong? If so, why?

Reply:
I think this behavior is confusing and unexpected. Logically a filter list composed entirely of ignored values should be equivalent to a filter list that was never configured at all. It is especially confusing when a list with a single ignored value is considered as not configured, but multiple ignored values are considered configured.

This PR normalizes `IsConfigured` so that it only returns true if valid values have been included.